cerk/src/kernel/broker_event.rs

```diff
@@ -41,26 +53,84 @@ pub enum BrokerEvent {
     ///
     ConfigUpdated(Config, InternalServerId),
 
-    /// The IncommingCloudEvent event indicates to the receiver that a new CloudEvent has been received from the outside world.
+    /// The IncomingCloudEvent event indicates to the receiver that a new CloudEvent has been received from the outside world.
     /// The event is produced by an input port and is sent to the Kernel. The Kernel sends the same event to the router.
     ///
     /// # Arguments
     ///
     /// * `InternalServerId` - id of the component which received the CloudEvent
+    /// * `CloudEventMessageRoutingId` - the unique identifier of the CloudEvent routing attempt
     /// * `CloudEvent` - the deserialized CloudEvent that the component has received
+    /// * `CloudEventRoutingArgs` - routing arguments to define how a CloudEvent should be routed
+    ///
+    IncomingCloudEvent(
+        InternalServerId,
+        CloudEventMessageRoutingId,
+        CloudEvent,
+        CloudEventRoutingArgs,
+    ),
```

Comment:
would it make sense to put these fields into a struct? I think it would be easier to work with later on (I had to check the docstrings a lot).
```
struct IncomingCloudEventPayload {
  pub message_routing_id: CloudEventMessageRoutingId,
  pub event: CloudEvent,
  pub source_id: InternalServerId,
  pub routing_args: CloudEventRoutingArgs,
}
```